VocaDbModel/Service/VideoServices/VideoServiceBilibili.cs

```diff
@@ -50,6 +50,14 @@ public override async Task<VideoUrlParseResult> ParseByUrlAsync(string url, bool
 			return VideoUrlParseResult.CreateError(url, VideoUrlParseResultType.LoadError, new VideoParseException($"Unable to load Bilibili URL: {x.Message}", x));
 		}
 
+		// cannot view video
+		// web returns the error message "啊叻？视频不见了？" (あれ? the video's gone?)
+		// api simultaneously returns the message "稿件不可见" (upload cannot be viewed)
+		if (response.Code == 62002)

```

Comment:
This check should also be changed to `if (response == null || response.Data == null || response.Code == 62002)`, because now both the response object and the `response.Data` object could be null.